bioio/writers/__init__.py

```diff
@@ -1,15 +1,21 @@
 #!/usr/bin/env python
-# -*- coding: utf-8 -*-
-
-from .ome_tiff_writer import OmeTiffWriter
-from .ome_zarr_writer_v2 import OmeZarrWriter as V2OmeZarrWriter
-from .ome_zarr_writer_v3 import OMEZarrWriter as V3OmeZarrWriter
-from .ome_zarr_writer_v3 import default_axes, downsample_data
-
-__all__ = [
-    "V3OmeZarrWriter",
-    "V2OmeZarrWriter",
-    "OmeTiffWriter",
-    "default_axes",
-    "downsample_data",
-]
+
+import sys
+from typing import List
+
+# Choose the right entry_points based on Python version
+if sys.version_info >= (3, 10):
+    from importlib.metadata import entry_points as _entry_points
+else:
+    from importlib_metadata import entry_points as _entry_points
```

Comment:
these lines (8 and 10) seem identical?

Reply:
resolved in [c4f40b1](https://github.com/bioio-devs/bioio/pull/126/commits/c4f40b166fac3544409d4537935bee62f29cc089)